docs/docs/legal/data-processing-agreement.mdx

```diff
@@ -1,113 +1,276 @@
 
-This Annex to the Agreement governs the Parties' data protection obligations in addition to the provisions of the Agreement.
+All capitalized terms not defined in this DPA will have the meanings set forth in the Agreement.
+Any privacy or data protection related clauses or agreement previously entered into by ZITADEL and the Customer, with regards to the subject matter of this DPA, will be superseded by and replaced with this DPA.
+No one other than a Party to this DPA, their successors and permitted assignees will have any right to enforce any of its terms.
 
-## Subject matter, duration, nature and purpose of the processing as well as the type of personal data and categories of data subjects
+This Addendum shall become legally binding upon Customer entering into the Agreement.
 
-This annex reflects the commitment of both parties to abide by the applicable data protection laws for the processing of Personal Data for the purpose of Processor's execution of the Framework Agreement.
+## Definitions
 
-The duration of the Processing shall correspond to the duration of the Agreement, unless otherwise provided for in this Annex or unless individual provisions obviously result in obligations going beyond this.
+"**Applicable Data Protection Law**" means all worldwide data protection and privacy laws and regulations applicable to the Personal Data, including, where applicable, EU/UK Data Protection Law and US Data Protection Laws (in each case, as amended, adopted, or superseded from time to time).
 
-In particular, the following Personal Data are part of the processing:
-<PiidTable />
+“**Controller**,” “**collecting**,” “**processor**,” and “**processing**,” shall have the meanings given to them under Applicable Data Protection Law.
 
-## Scope and responsibility
+“**Business**,” “**service provider**,” “**contractor**,” “**selling**,” “**sharing**” and “**third party**” shall have the meanings given to them under applicable US Data Protection Laws.
 
-Under this Agreement, the Processor shall process Personal Data on behalf of the Customer.
+"**Customer Data**" means information, data and other content, in any form or medium, that is submitted, posted or otherwise transmitted by or on behalf of the Customer through the ZITADEL Cloud or Services. For the avoidance of doubt, Customer Data includes Customer Personal Data.
 
-This Annex applies to all processing of Customer's data (including data of the users of Customer's organization) with reference to persons ("**Personal Data**") which is related to the Agreement and which is carried out by the Processor, its employees or agents.
+“**Customer Personal Data**” means, in any form or medium, all Personal Data that is processed by ZITADEL or its sub-processors on behalf of Customer in connection with the Agreement.
```

Comment:
> ... with the Agreement.

Which agreement is meant here? The DPA or the Framework or the Contract?

Reply:
Agreement is capital letters and defined. It's the Framework Agreement.